FeedAPIChallenge/Feed API/RemoteFeedLoader.swift

```diff
@@ -19,6 +19,66 @@ public final class RemoteFeedLoader: FeedLoader {
 	}
 	
 	public func load(completion: @escaping (FeedLoader.Result) -> Void) {
+		client.get(from: url) { [weak self] result in
+			
+			guard let _ = self else { return }
+			
+			switch result {
+			case .failure: completion(.failure(Error.connectivity))
+			case .success(let successResult):
+				
+				let response = successResult.1
+				let data = successResult.0
+				
+				let mapper = RemoteFeedMapper()
+				completion(mapper.map(response, data: data))
+			}
+		}
+	}
+}
+
+private struct RemoteFeedMapper {
+	private static let SuccessResponseCode = 200
+	
+	struct FeedRoot: Decodable {
+		let items: [FeedImageDTO]
+	}
+	
+	struct FeedImageDTO: Decodable {
+		enum Keys: String, CodingKey {
```

Comment:
To decode properties using `Decodable` the enum must be named `CodingKeys`.

```suggestion
		enum CodingKeys: String, CodingKey {
```